.github/workflows/xtest.yml

```diff
@@ -19,6 +19,10 @@ on:
         required: false
         type: string
         default: main
+      tests-ref:
```

Comment:
Can you please help me understand an example of how this is utilized by callers of the workflow?

Reply:
It seems like we'd always want to run this against `main` in consumer repos and on the PR branch in this repo?

Reply:
With circular test dependencies, it allows developers to see which test changes resolve failures in other repos. For example, the xtests never failed in this PR; however, they only passed in my platform PR after setting the [tests-ref](https://github.com/opentdf/platform/pull/1590) parameter.